lms/static/js/learner_dashboard/views/program_list_header_view.js

```diff
@@ -0,0 +1,58 @@
+import Backbone from 'backbone';
+
+import HtmlUtils from 'edx-ui-toolkit/js/utils/html-utils';
+
+import AlertListView, { mapAlertTypeToAlertHOF } from './program_alert_list_view';
+
+import programListHeaderTpl from '../../../templates/learner_dashboard/program_list_header_view.underscore';
+
+class ProgramListHeaderView extends Backbone.View {
+  constructor(options) {
+    const defaults = {
+      el: '.js-program-list-header',
+    };
+    super(Object.assign({}, defaults, options));
+  }
+
+  initialize(options) {
+    this.tpl = HtmlUtils.template(programListHeaderTpl);
+    this.data = options.context;
+    this.alertCollection = new Backbone.Collection(
+      // TODO: get this from api
+      this.data.programsData
+        .map((programData) =>
+          [
+            { type: 'no_enrollment' },
+            { type: 'subscription_trial_expiring' },
+          ].map(
+            mapAlertTypeToAlertHOF('program_list', programData, {
```

Comment:
please define this mock data some api level so it can be reused elsewhere.

Reply:
makes sense, I made the changes :)